src/components/ImageDetails/ImageDetails.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'useState' is defined but never used.